code/modules/mob/living/basic/space_fauna/spider/spider_abilities/hivemind.dm

```diff
@@ -61,10 +61,36 @@
 /datum/action/command_spiders/proc/spider_command(mob/living/user, message)
 	if(!message)
 		return
-	var/my_message = span_spider("<b>Command from [user]:</b> [message]")
+	var/my_message = format_message(user,message)
 	for(var/mob/living/basic/spider as anything in GLOB.spidermobs)
 		to_chat(spider, my_message)
 	for(var/ghost in GLOB.dead_mob_list)
 		var/link = FOLLOW_LINK(ghost, user)
 		to_chat(ghost, "[link] [my_message]")
 	user.log_talk(message, LOG_SAY, tag = "spider command")
+
+/datum/action/command_spiders/proc/format_message(mob/living/user, message)
+	return span_spiderbroodmother(("<b>Command from [user]:</b> [message]"))
+
+/**
+ * Sends a small message to all currently living spiders.
+ */
+/datum/action/command_spiders/communication_spiders
+	name = "Communication"
+	desc = "Send a command to all living spiders."
+	button_icon = 'icons/mob/actions/actions_animal.dmi'
+	button_icon_state = "message"
+	background_icon_state = "bg_alien"
+	overlay_icon_state = "bg_alien_border"
+	check_flags = AB_CHECK_CONSCIOUS
+
+/**
+ * Sends a message to all spiders from the target.
```

Comment:
This comment is copy/pasted from a different proc, and isn't true.

Reply:
I thought it sent a small message to all spiders its meant to do that does it not?